### PR TITLE
iOS: Update log prefix for device logs. We don't use SkyShell anymore

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

**Comment by chinmaygarde** [comment]
Fixes https://github.com/flutter/flutter/issues/868

File: packages/flutter_tools/lib/src/ios/device_ios.dart [hunk 1]
```diff
@@ -235,7 +235,7 @@ class IOSDevice extends Device {
       return 2;
     }
     return await runCommandAndStreamOutput([loggerPath],
-        prefix: 'iOS dev: ', filter: new RegExp(r'.*SkyShell.*'));
+        prefix: 'iOS: ', filter: new RegExp(r'.*FlutterRunner.*'));
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/ios/device_ios.dart]: The `.*` shouldn't be needed, it's implicit in a regular expression. And if your regular expression is just a string literal, you can replace it with just a string. (Assuming this is being used as a general match and not an anchored match.)

**Comment by Hixie** [comment]
lgtm

File: packages/flutter_tools/lib/src/ios/device_ios.dart (hunk 1 above)
Comment (chinmaygarde) [inline packages/flutter_tools/lib/src/ios/device_ios.dart]: The `filter` named argument expects a `RegExp`. So I couldn't specify a string. Deleted the `.*` though.

**Comment by chinmaygarde** [comment]
Done. Rebased.

**Pull request merged**